pkgs/top-level/all-packages.nix

```
@@ -35435,4 +35435,6 @@ with pkgs;
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
 
   mictray = callPackage ../tools/audio/mictray { };
+
+  swaysettings = callPackage  ../tools/system/swaysettings { };
```

```suggestion
  swaysettings = callPackage  ../tools/system/swaysettings { };
```

This is not a system tool.
This is an application, it should be in `../applications/misc`.

---

Ok moved